File: Easy/Palindrome/palindromeNum.dart
//  traduzido para Dart

bool isPalindrome(x) {
  var number = x;
  if (number.isNegative) {
    return false;
  } else {
    var numTXT = number.toString();
    numTXT = numTXT.split('').reversed.join();

    print("$numTXT $number");

    var result = (numTXT == number.toString()) ? true : false;
    return result;
  }
}

void main() {
  print(isPalindrome(10));
}

File: Easy/Palindrome/palindromeNum.js
//  infelizmente tive que criar pois por algum motivo o javascript não me deixava fazer direto da function isPalindrome.
function reverseString(str) {
    return str.split("").reverse().join("");
}

var isPalindrome = function(x) {
   var number = x
   if (Math.sign(number) == -1) {
       return false
   } else {
       var numTXT = number.toString()
        numTXT = reverseString(numTXT);
        
        var result = (numTXT == number) ? true : false
        return result
   }
};

console.log(isPalindrome(10))